src/index.js

```diff
@@ -34,7 +34,7 @@ app.use(router)
 app.use(noPathHandler)
 app.use(errorHandler)
 
-const httpServer = app.listen(server.port, () => {
+const httpServer = app.listen(server.port, '0.0.0.0', () => {
```

Comment:
delete '0.0.0.0', it's only used when we're trying to connect locally from another device